entities/src/main/java/org/n52/series/db/beans/parameter/ParameterFactory.java

```diff
@@ -120,7 +120,13 @@ public static ParameterEntity<?> from(DescribableEntity entity, ValueType valueT
         } else if (entity instanceof PlatformEntity) {
             entityType = EntityType.PLATFORM;
         }
-        return entityType != null ? from(entityType, valueType) : null;
+        if (entityType != null) {
+            ParameterEntity<?> e = from(entityType, valueType);
+            e.setDescribeableEntity(entity);
+            return e;
+        } else {
+            return null;
+        }
```

Comment:
remove else and `return null;` directly

Reply:
implemented in https://github.com/52North/sensorweb-server-db-model/pull/132/commits/44b891c1002474122a6ea1801cd222d51080310d